packages/core/patternlab-config.json

```diff
@@ -70,8 +70,11 @@
   },
   "patternExtension": "mustache",
   "patternStateCascade": ["inprogress", "inreview", "complete"],
-  "patternExportDirectory": "./pattern_exports/",
+  "patternExportAll": false,
+  "patternExportDirectory": "pattern_exports",
   "patternExportPatternPartials": [],
+  "patternExportPreserveDirectoryStructure": true,
```

Comment:
WOAH - really?! 🎉

Reply:
the code looked a bit scary to me, but it works in my initial testing, and currently only does the top-level.

the OP mentioned this is a bit of WIP, so it could be extended for sure